frigg/api/views.py

```diff
@@ -27,6 +28,11 @@ class BuildViewSet(viewsets.ModelViewSet):
     filter_backends = [BuildPermissionFilter]
     permission_classes = ReadOnly, permissions.DjangoModelPermissionsOrAnonReadOnly
 
+    def get_by_owner_name_build_number(self, request, owner, name, build_number):
+        return Response(BuildSerializer(Build.objects.get(project__owner=owner,
```

Comment:
I think it is necessary to have `Build.objects.permitted(request.user).get(...)` to avoid giving access to private projects. Also, it may be good with a test to check that `/api/builds/frigg/frigg/1/` returns 404.